public/index.html

```diff
@@ -1,18 +1,42 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <!-- Global site tag (gtag.js) - Google Analytics -->
-    <script async src="https://www.googletagmanager.com/gtag/js?id=G-P46DQLKKXK"></script>
+    <!-- Google Tag Manager -->
     <script>
-      window.dataLayer = window.dataLayer || [];
-      function gtag() {
-        dataLayer.push(arguments);
+      function gtm(w, d, s, l, i, auth) {
+        w[l] = w[l] || [];
+        w[l].push({ 'gtm.start': new Date().getTime(), event: 'gtm.js' });
+        var f = d.getElementsByTagName(s)[0],
+          j = d.createElement(s),
+          dl = l != 'dataLayer' ? '&l=' + l : '';
+        j.async = true;
+        j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl + auth;
+        f.parentNode.insertBefore(j, f);
       }
-      if (window.location.hostname.match(/^(www.)?terraware.io$/i)) {
-        gtag('js', new Date());
-        gtag('config', 'G-P46DQLKKXK');
+      if (w.location.hostname.match(/^(www.)?staging.terraware.io$/i)) {
+        gtm(
+          window,
+          document,
+          'script',
+          'dataLayer',
+          'GTM-TNX8NWM',
+          '&gtm_auth=Iwp9NkwNGO7TelQuq94FhQ&gtm_preview=env-7&gtm_cookies_win=x'
+        );
+      } else if (w.location.hostname.match(/^(www.)?terraware.io$/i)) {
+        gtm(
+          window,
+          document,
+          'script',
+          'dataLayer',
+          'GTM-TNX8NWM',
+          '&gtm_auth=oxl4mxgvnvET8ov_75EuYA&gtm_preview=env-1&gtm_cookies_win=x'
+        );
+      } else {
+        // Track non-staging/prod environments
+        // gtm(window, document, 'script', 'dataLayer', 'GTM-TNX8NWM', '&gtm_auth=1591mjZEVL_pulZkOX4gWg&gtm_preview=env-11&gtm_cookies_win=x');
```

Comment:
remove?

Reply:
I left it for testing (or if we eventually add a third stage to our dev pipeline).  Any objections to leaving it for those reasons?

Reply:
No objections, we can certainly keep it. Might be good to clarify the comment .. "Uncomment this section to test non-staging/prod environnents" or something like that..